src/components/Channel/Vote.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState, useEffect } from 'react';
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+
+import usePlayers from '../../hooks/channel/usePlayers';
+
+import { API } from '../../constants/api';
+
+import ErrorBox from '../ErrorBox';
+
+export default function Vote({ onVotingEnd }) {
+  const {
+    query: { channelId },
+  } = useRouter();
+  const { players, error } = usePlayers(channelId);
+  const [playerId, setPlayerId] = useState('');
+
+  if (error) {
+    return <ErrorBox message={error.message} />;
+  }
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      onVotingEnd();
+    }, 10000);
+
+    return () => clearTimeout(timer);
+  }, []);
+
+  async function selectCharacter({ currentTarget }) {
+    setPlayerId(currentTarget.id);
+
+    try {
+      const response = await fetch(`${API.URL}/channel/${channelId}`, {
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          state: 'voting',
+          playerId: currentTarget.id,
+        }),
+      });
+
+      const { result, message } = await response.json();
+
+      if (result === 'error') {
+        throw new Error(message);
+      }
+    } catch (err) {
+      return <ErrorBox message={err.message} />;
+    }
+  }
+
+  return (
+    <Wrapper>
+      <h1 className="vote-title">최고의 캐릭터에게 투표해주세요</h1>
+      <span className="vote-subtitle">투표는 한 번만 참여 가능합니다.</span>
+      <div className="characters">
+        {players &&
+          players.map((player) => (
```

Comment:
```suggestion
        {
          players?.map((player) => ()
        }
```
optional chaining을 사용해도 같은 결과가 나올 것 같습니다!